.travis.yml

```diff
@@ -1,21 +1,27 @@
+language: ruby
+
+sudo: false
+
+script: bundle exec rake travis
+
+rvm:
+  - 1.9
+  - 2.0
+  - 2.1
+  - 2.2
+  - jruby-19mode
+  - jruby-head
+  
+gemfile:
+  - Gemfile
+  - gemfiles/Gemfile-edge
+  
 matrix:
-  include:
-    - rvm: 1.8.7
-      gemfile: gemfiles/Gemfile.1.8.7
-    - rvm: 1.9.2
-      gemfile: gemfiles/Gemfile.1.9.2+
-      env: COVERAGE=true
-    - rvm: 1.9.3
-      gemfile: gemfiles/Gemfile.1.9.2+
-      env: COVERAGE=true
-    - rvm: 2.0.0
-      gemfile: gemfiles/Gemfile.1.9.2+
-      env: COVERAGE=true
-    - rvm: 2.1.1
-      gemfile: gemfiles/Gemfile.1.9.2+
-      env: COVERAGE=true
-    - rvm: ree
-      gemfile: gemfiles/Gemfile.1.8.7
-    - rvm: jruby
-      gemfile: gemfiles/Gemfile.1.9.2+
-      env: COVERAGE=true
\ No newline at end of file
+  fast_finish: true
+  allow_failures:
+    - rvm: jruby-head
+    - rvm: 2.2
+
+env:
+  - secure: "LFBE3H1FUpPR4yBoMk0R3xGp6pNR/Dn2/PkoUtdoDiLluNL+2LQVQPFYmfWjPYlbsQa8KY94lVT7pP8lfePkXHEJ4CtjoKGVmQjRG4DTNCrSUTuQiw08tIHBiZJCTZHk8ZoAcQ2ac5Sg9JAxjTpjz2CO77yL1XNzlPlSjZObuAc="
+  - secure: "euxN9HsbO4tn3W3CXUgMxEQJrDK3kACPrRGbqXnYG55BS5gWiM1V7zpjkKSBNlLFYJQidrT63Jrwr4B/RNmZTl5aRoFdEyIqgsfQKrTENxp2c7E5eXNfXExfXx4rYOw2uXeWYrVsyk24XhOjIioQA/zXhMwXEmBXjJd1uAheRw4="
```

Comment:
That's what I had... but then the tests always failed. I'm trying to see if that works better but it's probably not the way to do it.